section2/exercises/methods.rb

```diff
@@ -4,28 +4,49 @@
 # `ruby section2/exercises/methods.rb`
 
 # Example: Write a method that when called will print your name:
-def print_name
-  p "Severus Snape"
+def print_name()
+  puts "Sidarth Bagawandoss"
 end
 
-print_name
+print_name()
 
 # Write a method that takes a name as an argument and prints it:
 def print_name(name)
-  # YOUR CODE HERE
+  puts "This is #{name}"
 end
 
-print_name("Albus Dumbledore")
+print_name("Harry Potter")
 
-# Write a method that takes in 2 numbers as arguments and prints 
+# Write a method that takes in 2 numbers as arguments and prints
 # their sum. Then call your method three times with different arguments passed in:
-# YOUR CODE HERE
+def add(x, y)
+  puts "The sum of x & y is: #{x} + #{y}"
+  return x + y
+end
+
+college = add(12, 10)
+puts college
```

Comment:
Challenge yourself: If you want the `add` function to also print the final return, is there a way to update your function so that this line (28) becomes obsolete?

Pro: Reduce number of lines if you don't have to explicitly call `puts college` and `puts license`
Con: You may want your function `add` to have a final return of an integer value instead of a print value